connector/spanmetricsconnector/connector.go

```diff
@@ -46,8 +46,15 @@ const (
 
 	metricNameLatency = "latency"
 	metricNameCalls   = "calls"
+
+	defaultUnit = "ms"
 )
 
+var unitDividers = map[string]int64{
```

Comment:
What do you think about putting this into a function? Having this at the package level means anything can mutate this map, and I think it's pretty cheap to map the `unitDividers` at runtime.

Reply:
done

Reply:
The map is still mutable because one could replace the function values at runtime.

I was thinking more along the lines of:
```
func getUnitDivider(s string) int64 {
	return map[string]int64{
		"s":  time.Second.Nanoseconds(),
		"ms": time.Millisecond.Nanoseconds(),
	}[s]
}
```

or

```
func getUnitDivider(s string) int64 {
	if s == "s" {
		return time.Second.Nanoseconds()
	}
	return time.Millisecond.Nanoseconds()
}
```

Reply:
That makes sense! Applied your suggestion.